app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -33,45 +33,9 @@
       %ul.nav.nav-pills.nav-stacked
-
-        - if role_allows(:feature => "ems_container_show") && !@record.ext_management_system.nil?
-          %li
-            = link_to("#{ui_lookup(:table => "ems_container")}: #{@record.ext_management_system.name}",
-                {:controller => "ems_container", :action => 'show', :id => @record.ext_management_system.id.to_s},
-                {:title => _("Show this pod's parent %s") % ui_lookup(:table => "ems_container")})
-        - if role_allows(:feature => "containers")
-          - num_containers = @record.number_of(:containers)
-          - if num_containers == 0
-            %li.disabled
-              = link_to("#{ui_lookup(:tables => "containers")} (0)", "#")
-          - else
-            %li
-              = link_to("#{ui_lookup(:tables => "containers")} (#{num_containers})",
-                  {:action => 'show', :id => @record, :display => 'containers'},
-                  :title => _("View %s") % ui_lookup(:table => "containers"))
-        - if role_allows(:feature => "container_node") && !@record.container_node.nil?
-          %li
-            = link_to("#{ui_lookup(:table => "container_node")}: #{@record.container_node.name}",
-                {:controller => 'container_node', :action => 'show', :id => @record.container_node},
-                :title => _("View %s %s") % [ui_lookup(:table => "container_node"), @record.container_node.name])
-        - if role_allows(:feature => "container_service_show_list")
-          - num_services = @record.number_of(:container_services)
-          - if num_services == 0
-            %li.disabled
-              = link_to("#{ui_lookup(:tables => "container_service")} (0)", "#")
-          - else
-            %li
-              = link_to("#{ui_lookup(:tables => "container_service")} (#{num_services})",
-                {:action => 'show', :id => @record, :display => 'container_services'},
-                  :title => _("Show Container Services"))
-        - if role_allows(:feature => "container_project_show") && !@record.container_project.nil?
-          %li
-            = link_to("#{ui_lookup(:table => "container_project")}: #{@record.container_project.name}",
-                {:controller => "container_project", :action => 'show', :id => @record.container_project.id.to_s},
-                :title => _("Show this pod's parent %s") % ui_lookup(:table => "container_project"))
-        - if role_allows(:feature => "container_replicator_show") && !@record.container_replicator.nil?
-          %li
-            = link_to("#{ui_lookup(:table => "container_replicator")}: #{@record.container_replicator.name}",
-                {:controller => 'container_replicator', :action => 'show', :id => @record.container_replicator},
-                :title => _("Show %s %s") % [ui_lookup(:table => "container_replicator"),
-                  @record.container_replicator.name])
+        = single_relationship_link(@record, :ems_container, "ext_management_system")
```

Comment:
Thank you @moolitayer logic in views are so hard to read... This is great.   Look at all of the deletes!  :heart_eyes: